src/example/Simplest.js

```diff
@@ -0,0 +1,71 @@
+import ActionManager from "../action/ActionManager.js";
+import DoubleJumpFilter from "./filters/DoubleJumpFilter.js";
+
+function main() {
+  // App specific setup of the ActionManager
+
+  let actionManager = new ActionManager();
+
+  // Load a custom filter
+  actionManager.loadFilter(DoubleJumpFilter, "/filter/double-jump");
+
+  // Load up the maps that bind inputs to filters and actions
+  actionManager
+    .loadActionMaps(
+      // Maps used while the mode is flat
+      "./flat-playing.map",
+      "./flat-paused.map",
+      "./flat-menu.map",
+      // Maps used while the mode is portal
+      "./portal-playing.map",
+      "./portal-paused.map",
+      "./portal-menu.map",
+      // Maps used while the mode is immersive
+      "./immersive-playing.map",
+      "./immersive-paused.map",
+      "./immersive-menu.map"
+    )
+    .then(maps => {
+      console.log("Maps are loaded", maps);
+    })
+    .catch(err => {
+      console.log("Could not load action maps", err);
+    });
+
+  /*
+	While the app is running, it has to provide logic for [de]activating maps as
+	the mode changes between flat, portal, and immersive and as the app changes state like from playing
+	to paused to using a config menu
+	*/
+  actionManager.activateMapsExclusive("/map/flat-playing");
```

Comment:
the same as before, the app should activate `playing` action and depending on the active mode, it will use `flat-playing` or `portal-playing` or `immersive-playing`.